lib/resources/r4/charge_item.yaml

```diff
@@ -0,0 +1,69 @@
+---
+name: Chargeitem
+field_name_base_url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem
+fields:
+
+  - name: code
+    required: 'Yes'
+    type: CodeableConcept
+    description: A code that identifies the charge, like a billing code.
+    action: terminology
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code
+    binding:
+      description: A code that identifies the charge, like a billing code.
+      terminology:
+      - display: Bill Codes Types
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/CodeSystem/BillCodes
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#bill-codes-types
+
+  - name: reason
+    required: 'No'
+    type: CodeableConcept
+    description: Describes why the event occurred in coded or textual form.
+    action: terminology
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason
+    binding:
+      description: Describes why the event occurred in coded or textual form.
+      terminology:
+      - display: Bill Codes Types
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/CodeSystem/BillCodes-ICD
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#bill-codes-types
+
+  - name: Modifier Extension
+    terminology_name: extension[x].valueCodeableConcept
+    required: 'No'
+    type: Extension
+    description: A code providing additional detail about a product or service.
+    url: https://fhir.cerner.com/millennium/r4/financial/charge_item/#extensions
+    binding:
+      description: The status of the Account within a specific workflow. This binding is used in <code>modifier</code> extensions.
```

Comment:
The first sentence doesn't seem to apply to these extensions. (Same for procedure and revenue code)

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/050ae5f567c5bc377157d50c3e1f7bcc228fd7e0#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR38), [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/050ae5f567c5bc377157d50c3e1f7bcc228fd7e0#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR51), and [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/050ae5f567c5bc377157d50c3e1f7bcc228fd7e0#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR64)